src/globals/scss/objects/_content-wrapper.scss

```diff
@@ -0,0 +1,20 @@
+@include exports("content-wrapper") {
+
+  // Example usage
+
+  // <div class="govuk-o-wrapper">
+  //   Breadcrumbs, Phase banners, Back links are placed in here.
+
+  //   <div class="govuk-o-content">
+  //       Wrapper for the main content of your page which applies padding to the top / bottom
+  //   </div>
+
+  // </div>
+
+  .govuk-o-content {
```

Comment:
Would be good to make this consistent with the filename (or make the filename consistent with this).